posts/2023-11-16-Rust-1.74.0.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title: "Announcing Rust 1.74.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.74.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, you can get 1.74.0 with:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`](https://www.rust-lang.org/install.html) from the appropriate page on our website, and check out the [detailed release notes for 1.74.0](https://github.com/rust-lang/rust/releases/tag/1.74.0) on GitHub.
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you might come across!
+
+## What's in 1.74.0 stable
+
+### Lint configuration through Cargo
+
+As proposed in [RFC 3389](https://rust-lang.github.io/rfcs/3389-manifest-lint.html),
+the `Cargo.toml` manifest now supports a `[lints]` table to configure the
+reporting level (forbid, deny, warn, allow) for lints from the compiler and
+other tools. So rather than setting `RUSTFLAGS` with `-F`/`-D`/`-W`/`-A`, which
+would affect the entire build, or using crate-level attributes like:
```

Comment:
I don’t know which is right, but I’m hoping someone looked up if this is supposed to be affect or effect…

Reply:
I think I got it right, as a verb like "change" or "act on".
https://www.merriam-webster.com/grammar/affect-vs-effect-usage-difference